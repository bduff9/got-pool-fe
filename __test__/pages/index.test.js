import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import IndexPage from '../../pages/index.js';

describe('IndexPage', () => {
	/**
	 * @type {import('enzyme').ShallowWrapper<any, any, any>} wrapper
	 */
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<IndexPage />);
	});

	it('renders', () => {
		expect(wrapper).toBe(true);
	});

	test.todo('add tests for IndexPage');

	it('matches snapshot', () => {
		const component = renderer.create(<IndexPage />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
