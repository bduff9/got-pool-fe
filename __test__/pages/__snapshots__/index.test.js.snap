// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage matches snapshot 1`] = `
<div>
  <div />
  <nav
    className="nav"
  >
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/dogs"
          onClick={[Function]}
        >
          Dogs
        </a>
      </li>
      <li>
        <a
          href="/dogs/shepherd"
          onClick={[Function]}
        >
          Only Shepherds
        </a>
      </li>
    </ul>
  </nav>
  <div>
    <h1>
      This is the Front Page.
    </h1>
    <h3>
      Random dog of the day:
    </h3>
    <img
      alt=""
    />
  </div>
</div>
`;
